src/main/java/com/amazonaws/services/neptune/cli/AbstractTargetModule.java

```diff
@@ -10,12 +12,18 @@
 import com.github.rvesse.airline.annotations.restrictions.Once;
 import com.github.rvesse.airline.annotations.restrictions.PathKind;
 import com.github.rvesse.airline.annotations.restrictions.Required;
+import org.apache.commons.lang.StringUtils;
 
+import javax.inject.Inject;
 import java.io.File;
 import java.io.IOException;
 import java.util.UUID;
 
+import static com.amazonaws.services.neptune.util.AWSCredentialsUtil.getSTSAssumeRoleCredentialsProvider;
+
 public abstract class AbstractTargetModule implements CommandWriter {
+    @Inject
```

Comment:
Why is `@Inject` needed here?

Reply:
It is used by `rvesse.airline` (CLI utility library) that it should inject all of the options from `CredentialProfileModule` into this class. There is a newer syntax for in the library for this which does not use the javax inject notation, however this is the notation currently used throughout the repo.